rust/src/lib/error.rs

```diff
@@ -38,6 +38,22 @@ impl NmstateError {
     pub fn msg(&self) -> &str {
         self.msg.as_str()
     }
+
+    pub(crate) fn port_overbook(
```

Comment:
This function is only used once. And other code of this project is using `NmstateError::new()`.
Please be consistent with other codes.

Reply:
Done